Filters/exclusions.txt

```diff
@@ -1,3 +1,5 @@
+! https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/929
+||pinpoll.com^
```

Comment:
I'm not sure about unlocking either. It looks like a tracker and it's used on many websites.
`EasyPrivacy` also blocks it.

Reply:
This is legal widget. They have subdomain which used for tracking - `||pa.pinpoll.com^`
![image](https://user-images.githubusercontent.com/8361299/162212498-ebf6ba19-b641-466b-936f-7cb10e1a4b1f.png)
Easylist blocks too much, when used as hosts filter.

Reply:
Hm, ok then.